kumascript/macros/WebkitBug.ejs

```diff
@@ -10,22 +10,5 @@ var linkLabel = mdn.replacePlaceholders(mdn.localString({
     "ru": "Баг WebKit $1$",
 }), [$0]);
 
-/* TODO: Re-enable when we have a fix for bug 765642
```

Comment:
I deleted this comment because [referenced bug](https://bugzilla.mozilla.org/show_bug.cgi?id=765642#c7) was closed as WONTFIX.